modular_skyrat/modules/deforest_medical_items/code/chemicals/demoneye.dm

```diff
@@ -139,14 +151,17 @@
 
 
 /// Hurts a random organ, if its 'really_bad' we'll vomit blood too
-/datum/reagent/drug/demoneye/proc/hurt_that_mans_organs(mob/living/carbon/our_guy, damage, really_bad)
+/datum/reagent/drug/demoneye/proc/hurt_that_mans_organs(mob/living/carbon/our_guy, damage, really_bad = FALSE)
 	if(really_bad)
 		our_guy.vomit(0, TRUE, FALSE, 1)
 	our_guy.adjustOrganLoss(
-		pick(ORGAN_SLOT_BRAIN,ORGAN_SLOT_APPENDIX,ORGAN_SLOT_LUNGS,ORGAN_SLOT_HEART,ORGAN_SLOT_LIVER,ORGAN_SLOT_STOMACH),
+		pick(organs_we_damage),
```

Comment:
The static list could've been in this proc instead, rather than being a list on the datum itself. I don't really see why we'd want to change it, that's why I think reducing the scope would make more sense.

Reply:
yeah alright